ckan/config/middleware/flask_app.py

```diff
@@ -106,9 +106,18 @@ def is_null_session(self, obj):
         if not is_null:
             # Beaker always adds these keys on each request, so if these are
             # the only keys present we assume it's an empty session
+
             is_null = (
-                sorted(obj.keys()) == [
-                    u"_accessed_time", u"_creation_time", u"_domain", u"_path"]
+                (
+                    # Beaker 0.11 (py3)
+                    sorted(obj.keys()) == [
+                        u"_accessed_time", u"_creation_time",
+                        u"_domain", u"_path"]
+                ) or (
```

Comment:
Could also use `in`, eg:

    sorted(obj.keys()) in [
        [u"_accessed_time", u"_creation_time", u"_domain", u"_path"],
        [u"_accessed_time", u"_creation_time"]
    ]